v2/nodejs/jwt/createJWT.mdx

```diff
@@ -0,0 +1,26 @@
+---
+id: createJWT
+title: createJWT
+hide_title: true
+---
+
+## `createJWT(payload, validity)`
```

Comment:
Should be validitySeconds here and below too?

Reply:
The signature had `validity` as a parameter in the type declaration, but `validitySeconds` does make more sense. Made the change here and in the NodeJS SDK PR.